create-venv/action.yaml

```
@@ -0,0 +1,26 @@
+name: "Create venv"
+description: "Create a temporary venv"
+inputs:
+  python-command:
+    description: "The Python to use for creating the environment"
+    required: false
+    default: "python"
+outputs:
+  python_path:
+    description: "The path to the virtual environment directory, not including the bin or scipts directory."
+    value: ${{ steps.create-venv.outputs.temp-venv-path }}
+runs:
+  using: "composite"
+  steps:
+    - name: Create venv
+      id: create-venv
+      shell: sh
+      run: |
+        ${{ inputs.python-command }} "${{ github.action_path }}/create_temporary_venv.py"
```

There's a weird issue that happens when using the github context version of paths instead of the version available on env when running jobs in a container. the env version actually works and is update to the correct value inside containers / github context values are not.

```suggestion
        ${{ inputs.python-command }} "${GITHUB_ACTION_PATH}/create_temporary_venv.py"
```

You can see this issue for more on this specific var https://github.com/actions/runner/issues/716
Might be worth testing this inside a container as well as one of the matrix options to make sure we dont break this in the future

---

Well now, ain't that a corner...  It's ok though, we weren't matrixed enough anyways.  Now to port this to all the other action tests...  https://github.com/Chia-Network/actions/pull/51/commits/1f38b86f5c9365db537935e92943f3f4c26d0d13, https://github.com/Chia-Network/actions/pull/51/commits/03d23e3ee33f9319fd3dc21feef7c13f60669d44, https://github.com/Chia-Network/actions/pull/51/commits/6c16eb5cd39f896207016daebad6349116610414

And here's your failure.

https://github.com/Chia-Network/actions/actions/runs/3527071241/jobs/5915724410#step:5:11
```
python: can't open file '/home/ubuntu/actions-runner/_work/actions/actions/./create-venv/create_temporary_venv.py': [Errno 2] No such file or directory
```

Suggested fix committed in https://github.com/Chia-Network/actions/pull/51/commits/32910158b7227dfc1d7643efe20658955bb273de and tests passing in https://github.com/Chia-Network/actions/actions/runs/3527105504.